cli/running/running.go

```diff
@@ -115,7 +114,8 @@ type providerImpl struct {
 
 // updateCtx updates cmdCtx according to the current contents of the cfg file.
 func (provider *providerImpl) updateCtx() error {
-	cliOpts, _, err := configure.GetCliOpts(provider.cmdCtx.Cli.ConfigPath)
+	cliOpts, _, err := configure.GetCliOpts(provider.cmdCtx.Cli.ConfigPath,
+		provider.cmdCtx.Integrity.Repository)
```

Comment:
Things that take `cmdCtx.Integrity` are mostly an artifact of getting rid of global variables. In case of `CollectInstances` this is necessary because of a call chain `CollectInstances -> collectInstancesFromAppDir -> loadInstanceConfig -> integrity.NewDataCollectorFactory`, where `integrity.NewDataCollectorFactory` must accept an `IntegrityCtx`, as it uses its private fields in substituted implementation in tt-ee